SpatialGDK/Documentation/content/directory-structure.md

```diff
@@ -1,22 +1,19 @@
 <%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
 
 # GDK for Unreal directory structure
-The table below contains the contents of the built GDK for Unreal after running `BuildGDK.bat`.
+The table below lists the contents of the Unreal GDK repository directory after running `Setup.bat`.
```

Comment:
`SpatialOS GDK for Unreal` ? :trollface: 

Also isn't the `repository structure` or just `repository`?